ulog/log.go

```diff
@@ -37,6 +36,10 @@ import (
 	"github.com/uber/jaeger-client-go"
 )
 
+const _contextLogger contextKey = iota
```

Comment:
I thought context key better to be unique. If anyone use 0 or used 0, value will be overwrote.